legacy-packages/title-block/KaizenDraft/TitleBlock/TitleBlock.tsx

```diff
@@ -95,7 +95,7 @@ class TitleBlock extends React.Component<Props, State> {
     const { surveyStatus } = this.props
     if (surveyStatus == undefined) return
 
-    let variant
+    let variant: React.ComponentPropsWithoutRef<typeof Tag>["variant"]
```

Comment:
Interesting. So it's possible to reach into some other component's props typing without explicitly exporting it from that component?

Reply:
Yeah i prefer to the alternative, I think this is more descriptive as to what the type means, i.e. we don't care what it is, just that it is the same as the one the component accepts.